ignite/handlers/time_profilers.py

```diff
@@ -0,0 +1,792 @@
+import functools
+from collections import OrderedDict
+from typing import Any, Callable, Dict, List, Mapping, Sequence, Tuple, Union, cast
+
+import torch
+
+import ignite.handlers.timing
```

Comment:
Can't we just import Timer like: `from ignite.handlers.timing import Timer` ?

Reply:
You are missing 4th solution ! 
By the way, why do not we move tests actually ?

Reply:
Can you please provide a hint for me?